### DISCUSSION TITLE
Remote control - share link

### DISCUSSION THREAD

**Discussion by kyoushinosensei** [lllyasviel/Fooocus#1070] [Q&A]
Hello everyone.)) Please tell me where write in code or parameters and what to do to get a public link?

**Comment by Phrixus2023** [comment]
webui.py bottom
share=True,

**Reply by s-step** [reply]
tell me in more detail . I wrote it in the file webui.py bottom
share=True but at startup it still opens locally and there is no link in the console

**Reply by Phrixus2023** [reply]
shared.gradio_root.launch(
    inbrowser=args_manager.args.auto_launch,
    server_name=args_manager.args.listen,
    server_port=args_manager.args.port,
    share=True,
    auth=check_auth if args_manager.args.share and auth_enabled else None,
    blocked_paths=[constants.AUTH_FILENAME]
)

**Comment by s-step** [comment]
> shared.gradio_root.launch( inbrowser=args_manager.args.auto_launch, server_name=args_manager.args.listen, server_port=args_manager.args.port, share=True, auth=check_auth if args_manager.args.share and auth_enabled else None, blocked_paths=[constants.AUTH_FILENAME] )

A link is being created for the Internet. but the page opens and there is a white screen. Probably you still need to register a command --listen но вот куда ? 

in the assembly from automatics, I had it written like this set COMMANDLINE_ARGS= --opt-sdp-attention --upcast-sampling --listen --share

**Comment by yesbroc** [comment]
you can just add the --share flag

**Reply by vicento** [reply]
where should we add this flag ?

**Reply by Phrixus2023** [reply]
run.bat
